reference/api/batches.mdx

```diff
@@ -48,7 +48,12 @@ The `/batches` route gives information about the progress of batches of [asynchr
     }, 
     "progressTrace": { … },
     "writeChannelCongestion": { … },
-    "internalDatabaseSizes": { … }
+    "internalDatabaseSizes": { … },
+    "embedderRequests": {
+      "total": 12,
+      "failed": 5,
+      "lastError": "runtime error: received internal error HTTP 500 from embedding server\n  - server replied with `{\"error\":\"Service Unavailable\",\"text\":\"will_error\"}`"
```

Comment:
This snippet came from a test, it's not need it would just confuse readers

```suggestion
      "lastError": "runtime error: received internal error HTTP 500 from embedding server\n  - server replied with `{\"error\":\"Service Unavailable\"}`"
```